crates/telio-model/src/features.rs

```diff
@@ -1,11 +1,10 @@
 //! Object descriptions of various
 //! telio configurable features via API
 
-use std::{collections::HashSet, fmt, time::Duration};
+use std::{collections::HashSet, fmt};
 
 use num_enum::{IntoPrimitive, TryFromPrimitive};
 use serde::{de::IntoDeserializer, Deserialize, Deserializer, Serialize};
-use serde_with::{serde_as, DurationSeconds};
```

Comment:
Please check if serde_with can be removed from the dependencies

Reply:
It still used in `telio-crypto`, but I removed it everywhere else